test/e2e-multicluster-test/multicluster_cli_test.go

```diff
@@ -113,14 +116,46 @@ var _ = Describe("Test multicluster CLI commands", func() {
 			for _, format := range []string{"inline", "wide", "table", "list"} {
 				outputs, err := execCommand("status", app.Name, "-n", namespace, "--tree", "--detail", "--detail-format", format)
 				Expect(err).Should(Succeed())
-				Expect(string(outputs)).Should(SatisfyAll(
+				Expect(outputs).Should(SatisfyAll(
 					ContainSubstring("alias-worker-tree"),
 					ContainSubstring("Deployment/exec-podinfo"),
 					ContainSubstring("updated"),
 					ContainSubstring("1/1"),
 				))
 			}
 		})
+
+		It("Test vela logs", func() {
+			var (
+				err         error
+				session     *gexec.Session
+				waitingTime = 2 * time.Minute
+			)
+			podViewCMFile, err := os.ReadFile("./testdata/view/component-pod-view.yaml")
+			Expect(err).Should(BeNil())
+			podViewCM := &v1.ConfigMap{}
+			Expect(yaml.Unmarshal(podViewCMFile, podViewCM)).Should(BeNil())
+			Expect(k8sClient.Create(hubCtx, podViewCM)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
+
+			stopChannel := make(chan struct{}, 1)
+			defer func() {
+				stopChannel <- struct{}{}
+			}()
+
+			command := exec.Command("vela", "logs", app.Name, "-n", namespace, "--cluster", WorkerClusterName)
```

Comment:
why vela logs relates with pod-view velaql?

Reply:
vela logs need to query VelaQL components-pod-view to get all pod that belongs to this app